CI/appveyor/install_deps_msvc.sh

```diff
@@ -43,7 +43,6 @@ __mv_to_build_dir() {
 
 git clone https://github.com/analogdevicesinc/libiio "$SRC_LIBIIO"
 cd "$SRC_LIBIIO"
-git checkout 85bf9cd32138539252ed01c355cf766612cf47c9
```

Comment:
Better keep a hash here - otherwise you'll always pull the latest master, which may have bugs.